podman_hpc/podman_hpc.py

```diff
@@ -195,21 +146,35 @@ def get_params(args):
     return comm, image
 
 
+def read_confs():
+
+    mdir = os.environ.get(_MOD_ENV, "/etc/podman_hpc/modules.d")
+    confs = {}
+    for d in glob(f"{mdir}/*.yaml"):
+        conf = yaml.load(open(d), Loader=yaml.FullLoader)
+        confs[conf['name']] = conf
+    return confs
+
+
+def add_args(parser, confs):
+    for k, v in confs.items():
+        parser.add_argument("--%s" % (v["cli_arg"]), action="store_true",
+                            help=v.get("help"))
+
+
 def main():
     parser = argparse.ArgumentParser(prog='podman-hpc', add_help=False)
-    parser.add_argument("--gpu", action="store_true",
-                        help="Enable gpu support")
-    parser.add_argument("--mpich", action="store_true",
-                        help="Enable mpich support")
-    parser.add_argument("--hpc", action="store_true",
-                        help="Enable hpc support")
     parser.add_argument("--additional-stores", type=str,
                         help="Specify other storage locations")
     parser.add_argument("--squash-dir", type=str,
                         help="Specify alternate squash directory location")
     parser.add_argument("--update-conf", action="store_true",
```

Comment:
could we add any kind of "verbose" or "debug" option so the podman run statement gets printed somewhere?

Reply:
Good idea.  I'll make an issue for that.